src/webportal/src/app/job-submission/components/submission-section.jsx

```diff
@@ -160,13 +164,25 @@ export const SubmissionSection = (props) => {
       updatedTaskRoles,
       updatedParameters,
       updatedSecrets,
+      updatedExtras,
     ] = getJobComponentsFromConfig(updatedJob, {vcNames});
 
+    if (extras.tensorBoard) {
```

Comment:
```js 
if (extras.tensorBoard) {
      const updatedTensorBoardExtras = updatedExtras.tensorBoard || {};
      if (!isValidateTensorBoardExtras(updatedTensorBoardExtras )) {
          updatedExtras.tensorBoard = extras.tensorBoard;
      }
 }
```

wrap these sentences into a function `isValidateTensorBoardExtras` ?

Reply:
Got it. `isValidTensorBoardExtras` was added in `util.js`. But there are two parameters, `originalTensorBoardExtras` and `updatedTensorBoardExtras`. Because this func need to check if `randomStr` is modified.

Reply:
OK... If it has two parameters, maybe we should change the function name, such as: `validateUpdatedTensorBoardExtras` or something else. Just a suggestion, you can decide by yourself.

Reply:
Got it.